src/components/ui/PressCardList/Item/PressCardItem.module.css

```
@@ -0,0 +1,75 @@
+.root {
+  margin-bottom: 50px;
+  display: flex;
+  flex-direction: column;
+  justify-content: space-between;
+  gap: 24px;
+  position: relative;
+}
+
+.header,
+.body {
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
+}
+
+.body {
+  gap: 30px;
+}
+
+.date {
+  font-weight: 500;
+  font-size: 12px;
+  letter-spacing: 0.04em;
+  text-transform: uppercase;
+  color: #aaa078;
+}
+
+.title {
+  font-weight: 600;
+  font-size: 20px;
+  letter-spacing: -0.01em;
+  text-transform: uppercase;
+  color: #f8f7f0;
+}
+
+.imgWrapper {
+  width: 100%;
+  height: 290px;
+}
+
+.img {
+  width: 100%;
+  object-fit: cover;
+  aspect-ratio: 3/2;
+  object-position: center top;
+}
+
+@media screen and (min-width: 768px) {
+  .root {
+    flex-basis: calc((100% - 24px) / 2);
+    margin-bottom: 0;
+  }
+}
+
+@media screen and (min-width: 1180px) {
+  .root {
+    flex-basis: calc((100% - (68px * 2)) / 3);
+  }
+
+  .outline {
+    min-height: 115px;
+  }
+
+  .root:not(:first-of-type)::before {
+    content: '';
+    width: 1px;
+    height: 100%;
+    display: block;
+    background-color: rgba(125, 114, 97, 0.5);

```

@Yura33-dev We can also use our CSS variable here for better consistency. Since our variables are defined with just RGB values, it makes it easy to apply different opacities.

```suggestion
    background-color: rgb(var(--rgb-inactive), 0.5);
```